cs/cli/vw_arguments.h

```diff
@@ -36,11 +36,16 @@ public ref class VowpalWabbitArguments
   float m_power_t;
 
   internal : VowpalWabbitArguments(VW::workspace* vw)
-      : m_data(gcnew String(vw->data_filename.c_str()))
-      , m_finalRegressor(gcnew String(vw->final_regressor_name.c_str()))
+      : m_finalRegressor(gcnew String(vw->final_regressor_name.c_str()))
       , m_testonly(!vw->training)
       , m_passes((int)vw->numpasses)
   {
+    if (vw->data_filenames.size() >= 1) { m_data = gcnew String(vw->data_filenames.front().c_str()); }
+    else
+    {
+      m_data = gcnew String("");
```

Comment:
Please throw rather than silently ignore multiple test files

Reply:
added throw just in case, but not entirely sure it can reach that state.